src/zoktai/any.md

```diff
@@ -2,55 +2,67 @@
 
 ## About this guide
 
+This guide assumes you are playing on the japanese 1.0 release of Boktai 2.  
+It will likely work fine for other releases but some strategies like RNG manipulation will need different timings.  
 Directions for paths and exits will use ↖️, ↗️, ↙️ and ↘️.
 
+### Terminology
+
+- RNG manip: Short for "Random Number Generator Manipulation". Used to influence the randomness within the game to make things happen that are normally reliant on chance.
+- Soft reset: Pressing A + B + Start & Select at the same time reboots the game. This is useful for RNG manips and is faster when loading a save after a death.
+- STRG and AGIL: Strength and Agility. The second and third status point options. They influence your physical damage, movement speed and defense.
```

Comment:
STRG and AGIL are the third and fourth status point options

Reply:
good catch, I even pulled out the game to make sure and still made that error